src/components/synchronization-page/OrganisationUnitPage.jsx

```diff
@@ -1,25 +1,70 @@
 import React from "react";
 import PropTypes from "prop-types";
 import i18n from "@dhis2/d2-i18n";
+import _ from "lodash";
+
 import {
     OrganisationUnitGroupModel,
     OrganisationUnitGroupSetModel,
     OrganisationUnitModel,
 } from "../../models/d2Model";
+import { getOrgUnitSubtree } from "../../logic/metadata";
 import GenericSynchronizationPage from "./GenericSynchronizationPage";
 
 export default class OrganisationUnitPage extends React.Component {
     static propTypes = {
         d2: PropTypes.object.isRequired,
     };
 
+    state = {
+        children: [],
+        metadataTableKey: Math.random(),
```

Comment:
> The only work-around possible is to use initialSelection and ask the table to re-render

Yep, at some point we should probably add some prop to ObjectsTable (`selection: string[]`) so it also works in controlled mode. For now, this workaround should do the trick, the only problem is that it makes some extra requests that could be avoided.